<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SerialBox</class>
 <widget class="QFrame" name="SerialBox">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>294</width>
    <height>328</height>
   </rect>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="captionBox">
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <layout class="QGridLayout" name="gridLayout">
            <item row="5" column="0">
             <widget class="QLabel" name="label_5">
              <property name="text">
               <string>Упр. потоком:</string>
              </property>
             </widget>
            </item>
            <item row="3" column="0">
             <widget class="QLabel" name="label_4">
              <property name="text">
               <string>Четность:</string>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QLabel" name="label">
              <property name="text">
               <string>Устройство:</string>
              </property>
             </widget>
            </item>
            <item row="4" column="0">
             <widget class="QLabel" name="label_6">
              <property name="text">
               <string>Стоповые биты:</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QComboBox" name="BaudBox"/>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>Скорость бит/с:</string>
              </property>
             </widget>
            </item>
            <item row="5" column="1">
             <widget class="QComboBox" name="FlowBox"/>
            </item>
            <item row="0" column="1">
             <widget class="QComboBox" name="PortBox"/>
            </item>
            <item row="3" column="1">
             <widget class="QComboBox" name="ParityBox"/>
            </item>
            <item row="4" column="1">
             <widget class="QComboBox" name="StopBitsBox"/>
            </item>
            <item row="2" column="1">
             <widget class="QComboBox" name="DataBitsBox"/>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>Биты данных:</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QPushButton" name="btn_default">
            <property name="text">
             <string>По умолчанию (9600 etc.)</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QVBoxLayout" name="buttons_layout"/>
          </item>
          <item>
           <layout class="QVBoxLayout" name="layout_SEND"/>
          </item>
          <item>
           <layout class="QHBoxLayout" name="layout_status"/>
          </item>
          <item>
           <layout class="QVBoxLayout" name="layout_bottom_LOG"/>
          </item>
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="layout_power">
          <item>
           <widget class="QPushButton" name="btn_power">
            <property name="minimumSize">
             <size>
              <width>33</width>
              <height>70</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>33</width>
              <height>70</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton#btn_power {
	image: url(:/power_knock/image/black/off.png);
	border-style: inset;
}
QPushButton#btn_power:checked {
	image: url(:/power_knock/image/green/on.png);
	border-style: inset;
}</string>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer_2">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="layout_right_LOG"/>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>PortBox</tabstop>
  <tabstop>BaudBox</tabstop>
  <tabstop>DataBitsBox</tabstop>
  <tabstop>ParityBox</tabstop>
  <tabstop>StopBitsBox</tabstop>
  <tabstop>FlowBox</tabstop>
  <tabstop>btn_default</tabstop>
  <tabstop>btn_power</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
